neurom/analysis/morphtree.py

```diff
@@ -300,6 +300,16 @@ def trunk_azimuth(tree, soma):
     return np.arctan2(vector[COLS.Z], vector[COLS.X])
 
 
+def partition(tree):
+    '''Ratio of maximum number of sections
+       over minimum number of sections at
+       each bifurcation point.
+    '''
+    return [np.max([n_sections(i.children[0]), n_sections(i.children[1])]) /
+            np.min([n_sections(i.children[0]), n_sections(i.children[1])])
```

Comment:
Ok, I was thinking to add this into morphmath if it makes sense and then call it from morphtree.